domain.md

```diff
@@ -0,0 +1,71 @@
+In order to deploy CAM that is integrated with customer own Active Directory (AD), the folloing requirements must be met. When deploying CAM, the **account credential and DC fqdn** must be provided, and the DC must be able to communicated with the given **fqdn**.
+
+# 1) Creating an account with delegated permissions to join a computer to domain. 
+
+The following steps setup an account with delegated privileges to join a computer to domain, assuming an account already created, otherwise, please create an account first.
+
+ * Login Domain Controller Server.
+ * Lanuch Active Directory Users and Computers.
+ * Locate the **OU** that you want to join a computer to and right-click and then click Delegate Control. 
```

Comment:
OU's are great but what should they do before we support OU's? Maybe the doc here is fine if we know what to do in the meantime - can you click on something more 'root' than the OU?

Reply:
yes, in the GUI, you could click on 'root' instead of one of OU. Based my understanding, it does not impact CAM deploy. If the delegation is under some 'OU', then when joining computer to domain using this account, the computer should join under this 'OU' automatically. CAM does not to know which OU the computer will join.

Reply:
Oh that's cool. Maybe you just solved another problem, which is great! :) But if 'root' is okay (what's it called in the GUI?) then would be good to say that.

Reply:
Active Directory Users and Computers